client/my-sites/email/email-providers-comparison/stacked/index.tsx

```diff
@@ -269,6 +325,35 @@ const EmailProvidersStackedComparison = ( {
 			{ ! isDomainInCart && domain && <EmailExistingPaidServiceNotice domain={ domain } /> }
 
 			<>
+				{ ! currentUserCanAddEmail && (
+					<PromoCard className="email-providers-stacked-comparison__owner-notice">
+						<p>
+							{ translate(
+								'An email solution can only be purchased by the domain owner. ' +
+									'To make a purchase, please {{link}}log in{{/link}} with the account that purchased the domain. ' +
+									"If you don't have access to that account, please reach out to the domain owner {{reachOutLink}}%(ownerUserName)s{{/reachOutLink}}",
+								{
+									components: {
+										link: <a href={ loginUrl } onClick={ () => onClickLink( 'user_login' ) } />,
```

Comment:
I would suggest tracking `owner_login` to be consistent with the other clicking tracking value of `owner_contact`. 

I also noticed a behaviour where the tracking call for login link appeared to have failed. This only happens when we're linking to a new page in the same browser tab, the tracking call results in `200 OK` status when the `<a>` target is set to `_blank`. Should we be worried about this? 

<img width="1379" alt="Screen Shot 2022-09-23 at 12 10 22 PM" src="https://user-images.githubusercontent.com/104910361/192005938-542f4e79-dc6b-4e18-9ebb-f8032a9109b8.png">